src/Microsoft.Azure.Relay/Common/TimeoutHelper.cs

```diff
@@ -133,22 +112,19 @@ public TimeSpan RemainingTime()
                 this.SetDeadline();
                 return this.originalTimeout;
             }
-            else if (this.deadline == DateTime.MaxValue)
+
+            if (this.deadline == DateTime.MaxValue)
```

Comment:
Please change this back to else if to avoid requiring the empty line between the if and this block in question. (fewer empty lines are generally preferable.)